FAQ.md

```diff
@@ -0,0 +1,66 @@
+# FAQ
+
+Answers to frequently asked questions about the [cleanvision](https://github.com/cleanlab/cleanvision) open-source package.
+
+The code snippets in this FAQ come from a fully executable notebook you can run via Colab or locally by downloading it [here](https://github.com/cleanlab/cleanlab/blob/master/docs/source/tutorials/faq.ipynb).
+
+
+1. **What kind of machine learning tasks can I use cleanvision for?**
+
+CleanVision is independent of any machine learning tasks as it directly works on images and does not require and labels or metadata to detect issues in the dataset. The issues detected by CleanVision are helpful for all kinds of machine learning tasks.
+
+2. **Can I check for specific issues in my dataset?**
+
+Yes, you can specify issues like `light` or `blurry` in the issue_types argument when calling `imagelab.find_issues`
+
+```python
+imagelab.find_issues(issue_types={"light": {}, "blurry": {}})
+```
+
+3. What dataset formats does CleanVision support?
+
+Apart from plain image files, CleanVision also works with HuggingFace and Torchvision datasets. You can use the dataset objects as is with the `image_key` argument.
+
+```python
+imagelab = Imagelab(hf_dataset=dataset, image_key="image")
+```
+
+
+
+For more detailed usage instructions and examples, please refer to the [official documentation](https://cleanvision.readthedocs.io/).
+
+## Known issues
+
+1. **Encountering error from `multiprocessing` library while running cleanvision.**
```

Comment:
is this a good name for the type of issue?

AFAIK this issue when it pops up makes little mention of multiprocess, and most users have no idea the root cause was multiprocessing. The way you phrased the FAQ is only useful to users who already know the root cause. Try to never write FAQs like that, instead write them based on the exact error message the users will see (or a generalized version that most users will encounter something similar to). 

Cf. how I renamed the original GH issue and I pinned it to make it more visible:

https://github.com/cleanlab/cleanvision/issues/230